### PR TITLE
Mandate

### PR DESCRIPTION
**Pull request by gcharest**
Had a quick meeting yesterday with @remyb2canada @jaysonmc and @smellems to try to clarify the mandate.

Please review to make sure it's properly set based on feedback.

Thanks!

### CODE REVIEW

File: TeamMandate/Mandate.md [hunk 1]
```diff
@@ -8,22 +8,15 @@ The [Canadian Experience](../Strategies/HumanDevelopmentLifeCycle.md) is when Ca
...
-This will be done through an open collaborative approach that will further ESDC's mandate in the digital era by aligning IITB with the Government of Canada's Digital Standards and Treasury Board's Policy on Service and Digital.
-
 ## Mandate (team's authority within IITB)
 
-- Define the vision of IT within ESDC
-- Briefing senior management on key organisational problems and recommended steps to address them
-- Present recommended course of action to any ESDC governance body
+- Diagnose key ESDC organisational problems
+- Identify recommended course of action to any ESDC governance body
+- Measure progress of strategies implementation against approved key performance
```
Comment (jaysonmc) [inline TeamMandate/Mandate.md]: I would add "ability to review information from groups within ESDC as they pertain to efficiency" or some such thing. 

ie. If we ask for information on how long someones process takes, and they don't give us access, we have something to point to that says we should in fact be granted access (granted it should be in the open anyways, but that's not the reality we work in ATM).

Comment (gcharest) [reply TeamMandate/Mandate.md]: Ah OK, I see! Trying to come up with a sentence simliar to the others.

**Comment by gcharest** [comment]
@gabrielcossette as-tu eu une chance de réviser?

File: TeamMandate/Mandate.md (hunk 1 above)
Comment (gabrielcossette) [reply TeamMandate/Mandate.md]: In my view, "Diagnose key ESDC organisational problems" gives us this authority.

**Pull request merged**